### PR TITLE
Templates UI

### PR DESCRIPTION
**Pull request by agriffard**
Related to #4747

### CODE REVIEW

**Comment by agriffard** [comment]
image

image

File: src/OrchardCore.Modules/OrchardCore.Templates/Views/_ViewImports.cshtml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 @inherits OrchardCore.DisplayManagement.Razor.RazorPage<TModel>
+@using Microsoft.Extensions.Localization;
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.Templates/Views/_ViewImports.cshtml]: Why the usings are here?!! only one view use them

Comment (agriffard) [reply src/OrchardCore.Modules/OrchardCore.Templates/Views/_ViewImports.cshtml]: So people don't have to wonder why T.Plural doesn't work if they use it in another view.

Comment (Skrypt) [reply src/OrchardCore.Modules/OrchardCore.Templates/Views/_ViewImports.cshtml]: This looks fine to me, no big deal.

**Comment by hishamco** [comment]
LGTM :shipit:

**Pull request merged**